AdyenActions/UI/View Controllers/Await/AwaitComponentViewModel.swift

```diff
@@ -36,12 +36,16 @@ internal struct AwaitComponentViewModel {
     /// - Parameter localizationParameters: The localization parameters to control some aspects of how strings are localized
     internal static func viewModel(with paymentMethodType: AwaitPaymentMethod,
                                    localizationParameters: LocalizationParameters? = nil) -> AwaitComponentViewModel {
+        let localizationKey = LocalizationKey(key: "adyen.\(paymentMethodType.rawValue).confirmPayment")
         switch paymentMethodType {
         case .mbway, .blik:
-            let localizationKey = LocalizationKey(key: "adyen.\(paymentMethodType.rawValue).confirmPayment")
             return AwaitComponentViewModel(icon: paymentMethodType.rawValue,
                                            message: localizedString(localizationKey, localizationParameters),
                                            spinnerTitle: localizedString(.awaitWaitForConfirmation, localizationParameters))
+        case .upicollect:
+            return AwaitComponentViewModel(icon: paymentMethodType.rawValue,
```

Comment:
So here spinnerTitle is `localizationKey` and in the above case it's the message. Is it intended this way?

Reply:
This is intentional.